cmake/Findspdlog.cmake

```diff
@@ -1,11 +0,0 @@
-find_package(PkgConfig)
-
-if(NOT EXISTS "${spdlog_INCLUDE_DIR}")
-    find_path(spdlog_INCLUDE_DIR
-            NAMES spdlog/spdlog.h
-            DOC "spdlog library header files"
-            )
-endif()
-
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(spdlog REQUIRED_VARS spdlog_INCLUDE_DIR)
```

Comment:
This file is useless since spdlog is being found in config mode. spdlogConfig.cmake is looked for instead which is installed with spdlog.